### ISSUE TITLE
Down > Up Retries Option

### ISSUE THREAD

**Issue by 1A3Dev** [louislam/uptime-kuma#3031]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

If something being monitored is having intermittent issues it causes inaccurate statuses due to the fact it instantly gets marked as Up if it's detected as online so I think it could make sense to have another "Retries" option however instead of checking Up > Down this would check Down > Up.

For example if a website is marked as Down it would then require X amount of heartbeats to return as online before actually marking it as Up.

### ✔️ Solution

Rename the current "Retries" option to "Down Retries" and add a new option for "Up Retries" which does the same as the existing one however checks Down > Up instead of Up > Down.

**Comment by CommanderStorm** [comment]
I would argue that this extra bookkeeping is mostly unnecessary.
In my experience, services mostly don't hover in the up-down state.
=> I would expect, you mostly want to get that notification that things are back okay as soon as possible after addressing the issue.

What is the exact use case you had a problem with `Retries`?

**Comment by 1A3Dev** [comment]
> What is the exact use case you had a problem with `Retries`?

I had a web server that was intermittently switching between HTTP 200 and 503 which was spamming notifications even with retries set to 2 which would be less spammy if this existed.

**Comment by CommanderStorm** [comment]
Okay, but that is something that should not happen => something you WANT to be notified for, right?

**Comment by 1A3Dev** [comment]
> Okay, but that is something that should not happen => something you WANT to be notified for, right?

No, the issue was caused by the hosting company where the web server was and I wanted to know it was down without getting a large amount of notifications.

**Comment by CommanderStorm** [comment]
I think this would be rather hard to explain. Fixing the things you monitor should be the goal and I hope you fixed your issue.

I am going to close this issue, unless there is a more legitemante claim (more than "the thing I am monitoring is broken, I don't care") to why something like this is nessesary or good.
If there is something I missed, we can reopen.
